src/app/components/normal-component/chat/chat.component.spec.ts

```diff
@@ -252,6 +254,22 @@ describe('ChatComponent', () => {
             const chatDiv: DebugElement = testUtils.findElement('#chatDiv');
             expect(component.scrollTo).toHaveBeenCalledWith(chatDiv.nativeElement.scrollHeight);
         }));
+        it('should not loadChatContent each time user is updated twice', fakeAsync(() => {
+            // Given a chat component, hence subscribed to the current user
```

Comment:
remove hence

Reply:
aren't all chat component subscribed to current user ?